cli/tests/test_rebase_command.rs

```diff
@@ -1229,6 +1295,554 @@ fn test_rebase_with_child_and_descendant_bug_2600() {
     "###);
 }
 
+#[test]
+fn test_rebase_revision_after() {
+    let test_env = TestEnvironment::default();
+    test_env.jj_cmd_ok(test_env.env_root(), &["init", "repo", "--git"]);
+    let repo_path = test_env.env_root().join("repo");
+
+    create_commit(&test_env, &repo_path, "a", &[]);
+    create_commit(&test_env, &repo_path, "b1", &["a"]);
+    create_commit(&test_env, &repo_path, "b2", &["a"]);
+    create_commit(&test_env, &repo_path, "c", &["b1", "b2"]);
+    create_commit(&test_env, &repo_path, "d", &["c"]);
+    create_commit(&test_env, &repo_path, "e", &["c"]);
+    create_commit(&test_env, &repo_path, "f", &["e"]);
+    // Test the setup
+    insta::assert_snapshot!(get_long_log_output(&test_env, &repo_path), @r###"
+    @  f  lylxulpl  88f778c5
+    ◉  e  kmkuslsw  48dd9e3f
+    │ ◉  d  znkkpsqq  92438fc9
+    ├─╯
+    ◉    c  vruxwmqv  c41e416e
+    ├─╮
+    │ ◉  b2  royxmykx  903ab0d6
+    ◉ │  b1  zsuskuln  072d5ae1
+    ├─╯
+    ◉  a  rlvkpnrz  2443ea76
+    ◉    zzzzzzzz  00000000
+    "###);
+    let setup_opid = test_env.current_operation_id(&repo_path);
+
+    let (stdout, stderr) = test_env.jj_cmd_ok(&repo_path, &["rebase", "-r", "c", "--after", "c"]);
+    insta::assert_snapshot!(stdout, @"");
+    // Rebasing a commit after itself should be a no-op.
+    insta::assert_snapshot!(stderr, @r###"
+    Skipping rebase of commits:
+      vruxwmqv c41e416e c | c
+      znkkpsqq 92438fc9 d | d
+      kmkuslsw 48dd9e3f e | e
+      lylxulpl 88f778c5 f | f
```

Comment:
Off topic: I wonder how often this is useful or if we should skip it

Reply:
Yeah it might not be that useful, perhaps just printing the number of commits which it skipped rebasing would as better?